requirements.txt

```diff
@@ -5,3 +5,4 @@ requests~=2.12
 pyyaml==3.12
 beautifulsoup4==4.6.0
 python-dotenv==0.7.1
+click==6.6 # Temporary fix for coala error with dependencies
```

Comment:
move it to test-dep

Reply:
hmm... yeah, it's related with test but I thought it should be near with the related library.

Reply:
``coala`` is the related lib here. I think it would be better at the top of the test-dep

Reply:
got it.